test/image.test.js

```diff
@@ -19,6 +21,30 @@ describe('Image', function () {
     assert.ok(Image instanceof Function);
   });
 
+  it('Image set src to JPEG', function () {
+    var img = new Image
+      , onloadCalled = 0;
+
+    assert.strictEqual(null, img.onload);
+    assert.strictEqual(false, img.complete);
+
+    img.onload = function () {
+      onloadCalled += 1;
+      assert.strictEqual(img.src, jpg_face);
+      assert.strictEqual(485, img.width);
+      assert.strictEqual(401, img.height);
+      assert.strictEqual(true, img.complete);
+    };
+
+    img.onerror = function (e) {
+      console.error("ERROR", e); // temporary...
```

Comment:
How about `onerrorCalled = 0`, `onerrorCalled += 1` and `assert.strictEqual(onerrorCalled, 0)`?